app/views/posts/show.html.erb

```diff
@@ -1 +1,15 @@
-<h1>Post 1</h1>
\ No newline at end of file
+<section class="posts-show">
+  <div class="post-header">
+    <h2>Post #<span><%= @post.id %></span> by <span><%= @user.name %></span></h2>
```

Comment:
_You did an excellent job, however, there are small places you need to work on._ 

- [ ] To make your code professional, Kindly remove commented code in this line

Reply:
That is not commented code @Rustamxon7

Reply:
@Rustamxon7 please reply, should I resubmit. I have already used up two code reviews and this was unfairly graded

Reply:
Hello, just a sec...

Reply:
<img width="807" alt="Screenshot 2022-12-01 at 11 58 54" src="https://user-images.githubusercontent.com/69011963/204986551-2fb6b764-ab95-414e-a89d-e7eb3db6f43a.png">

In my vscode this is shown as commented.

Reply:
Maybe just an extension man

Reply:
If you check the render, you can see the post.id and the @user.name

Comments in HTML normally start with <--comment-->

Reply:
Why would I comment out the closing </h2> tag? Do I resubmit?

Reply:
Yes, I agree with you. However, this is the guideline we need to follow. I'm reaching out to Senior Code reviewers. To get the full answer. Hope you will find some minutes :).

Reply:
![code3](https://user-images.githubusercontent.com/77661932/204988321-fd36da8f-9036-404f-8d6c-a75849f3afce.png)

Reply:
@Rustamxon7 please advise

Reply:
> If you check the render, you can see the post.id and the @user.name
> 
> Comments in HTML normally start with <!--comment-->

In HTML you are right, but in the .erb file, it accepts it as a comment.

Reply:
But you can see clearly in my vscode its not a comment

![code4](https://user-images.githubusercontent.com/77661932/204989650-5435c1d4-f9f0-4a8b-b519-e6b48ee16045.png)

Reply:
@Rustamxon7 I just added line no. 4; you can see it's highlighted as a comment. You are using an extension in your vscode that apparently takes anything starting with a # symbol as a comment. Hash \# symbols are used as comments in the ruby language and python.

Reply:
@itsmraga-hub, I have reached out to Seniors, and they accepted your request. Your code is totally fine. You are able to re-request another review without changing the (#)gs.